src/index.ts

```diff
@@ -381,13 +381,13 @@ export default class Tree<Key=number, Value=any> {
   }
 
```

Comment:
I wasn't sure how you wanted to update the docs to include the change, so I didn't update the readme. I'm open to suggestions